### PR TITLE
Http, Csv to Kafka: latest versions

### PR DESCRIPTION
**Pull request by ennru**
Use the latest library versions.
Use sbt 1.3.6.
Switch to the new `ActorSystem` API.
Drop some of the course management sbt features.

### CODE REVIEW

File: http-csv-to-kafka.conf [hunk 1]
```diff
@@ -1,8 +1,8 @@
 // https://github.com/lightbend-training/course-management-tools
 //
 // commands:
-// linearize -f -cfg http-csv-to-kafka.conf /Users/enno/dev/alpakka-samples /Users/enno/dev/alpakka-samples-lineraized
-// delinearize -cfg http-csv-to-kafka.conf  /Users/enno/dev/alpakka-samples /Users/enno/dev/alpakka-samples-lineraized/alpakka-samples
+// linearize -f -cfg http-csv-to-kafka.conf /Users/enno/dev/alpakka-samples /Users/enno/dev/alpakka-samples-linearized
+// delinearize -cfg http-csv-to-kafka.conf  /Users/enno/dev/alpakka-samples /Users/enno/dev/alpakka-samples-linearized/alpakka-samples
```
Comment (seglo) [inline http-csv-to-kafka.conf]: Is this file just for course management?  Can it be deleted?

File: alpakka-sample-http-csv-to-kafka/step_002_extract_http_entity/src/main/scala/samples/Main.scala [hunk 2]
```diff
@@ -5,22 +5,23 @@
...
-import akka.stream._
 import akka.stream.scaladsl.{ Sink, Source }
 import akka.util.ByteString
 
 import scala.concurrent.Future
 
 object Main extends App {
 
-  implicit val actorSystem = ActorSystem("alpakka-samples")
+  implicit val actorSystem: ActorSystem[Nothing] = ActorSystem[Nothing](Behaviors.empty, "alpakka-samples")
```
Comment (seglo) [inline alpakka-sample-http-csv-to-kafka/step_002_extract_http_entity/src/main/scala/samples/Main.scala]: You may have explained this, but if we're not using Akka Typed, why do we create a typed `ActorSystem`?

File: http-csv-to-kafka.conf (hunk 1 above)
Comment (ennru) [reply http-csv-to-kafka.conf]: Yes, it is required for the use with the Course Management tool.
We keep it to be able to linerarize/delineraize, even if that requires some special handling.

File: alpakka-sample-http-csv-to-kafka/step_002_extract_http_entity/src/main/scala/samples/Main.scala (hunk 2 above)
Comment (ennru) [reply alpakka-sample-http-csv-to-kafka/step_002_extract_http_entity/src/main/scala/samples/Main.scala]: We try to show the "new actor API" in all examples. Our other examples do so as well.

**Pull request merged**